core/domain/exp_services.py

```diff
@@ -237,6 +237,42 @@ def get_multiple_explorations_by_id(exp_ids, strict=True):
     return result
 
 
+def get_displayable_exploration_summaries_matching_ids(exploration_ids,
```

Comment:
We should distinguish between exploration summaries (objects) and exploration summary dicts. I suggest renaming this function to get_displayable_exp_summary_dicts_matching_ids(...)